distribution/project-manager/THIRD-PARTY/org.apache.tika.tika-core-2.4.1/NOTICE

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
No changes to `project-manager` in this PR please.

Reply:
Yeah, this was a bad merge or something

Reply:
@JaroslavTulach Please unblock the PR 🙏